java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -114,22 +120,23 @@ protected File getBinary() {
                 }
                 binary.setExecutable(true);
             } catch (Exception e) {
-                LOG.warning(String.format("%s getting Selenium Manager: %s",
-                        e.getClass().getSimpleName(), e.getMessage()));
+                throw new WebDriverException("Unable to obtain Selenium Manager", e);
             }
         }
         return binary;
     }
 
     public String getDriverPath(String driverName) {
+        if (!ImmutableList.of("geckodriver", "msedgedriver", "chromedriver").contains(driverName)) {
+            throw new WebDriverException("Unable to locate driver with name: " + driverName);
+        }
+
         String driverPath = null;
         File binaryFile = getBinary();
         if (binaryFile != null) {
             String output = runCommand(binaryFile.getAbsolutePath(),
                     "--driver", driverName.replaceAll(EXE, ""));
```

Comment:
I though the driver name in Windows contained the `.exe` at the of the end of `driverName`. But since this not like that (the previous condition you added is supposed to work also in Windows, right?), the `replaceAll(EXE, "")` in this line can be removed.

Reply:
hmm, maybe I'm wrong; I'll double check this in all the languages.

Reply:
Found a bug running this on Mac, so I'm going to pull this up to check Windows behavior.